CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Minor Changes
 
 - Added redundancy to `CoreNamespace.getTokensForOwner()` to handle failures when fetching token metadata.
+- Added support for the `acquiredAt` field for ordered `getNftsForOwner` queries.
```

Comment:
Add a comment that this is only available when specifying the orderby = transfertime.